components/eventing-controller/testing/bebmock.go

```diff
@@ -147,7 +190,9 @@ func GetSubscriptionResponse(m *BEBMock) ResponseWithName {
 	return func(w http.ResponseWriter, key string) {
 		subscriptionSaved := m.Subscriptions.GetSubscription(key)
 		if subscriptionSaved != nil {
-			subscriptionSaved.SubscriptionStatus = bebtypes.SubscriptionStatusActive
+			if subscriptionSaved.SubscriptionStatus == "" {
+				subscriptionSaved.SubscriptionStatus = bebtypes.SubscriptionStatusActive
+			}
```

Comment:
Why default to active if there is no status?

Reply:
Because, by default we want the success path, so EventMesh mock should mark its subscriptions ready when created by the controller.